### DISCUSSION TITLE
%E6%9C%AA%E5%88%86%E7%B1%BB%E9%95%BF%E7%AF%87/%E5%9C%A3%E5%A5%B3%E4%B9%8B%E8%B7%AF_%E4%BD%9C%E8%80%85%EF%BC%9A%E6%B7%B1%E6%B5%B7%E6%9C%B1%E7%BA%A2_page/

### DISCUSSION THREAD

**Discussion by giscus** [project-polymorph/trans-digital-cn#8] [Announcements]
# %E6%9C%AA%E5%88%86%E7%B1%BB%E9%95%BF%E7%AF%87/%E5%9C%A3%E5%A5%B3%E4%B9%8B%E8%B7%AF_%E4%BD%9C%E8%80%85%EF%BC%9A%E6%B7%B1%E6%B5%B7%E6%9C%B1%E7%BA%A2_page/

包含伪娘、男娘、性转、变性、CDTS 变身文学与跨性别/多元性别相关小说（非限制级）的内容， 主要关注剧情向内容，关注日常的性别转换相关的生活与剧情冲突。

https://novel.transchinese.org/%E6%9C%AA%E5%88%86%E7%B1%BB%E9%95%BF%E7%AF%87/%E5%9C%A3%E5%A5%B3%E4%B9%8B%E8%B7%AF_%E4%BD%9C%E8%80%85%EF%BC%9A%E6%B7%B1%E6%B5%B7%E6%9C%B1%E7%BA%A2_page/

**Comment by yunwei37** [comment]
据说很好
